client/web/src/hooks/useWebsocket.js

```diff
@@ -32,8 +32,8 @@ const useWebsocket = (onOpen, onMessage) => {
             // Generate the new host value with the same IP but different port
             var newHost = ipAddress + ':' + newPort;
 
-            const ws_path = ws_scheme + '://' + newHost + `/ws/${clientId}`;
-
+            const ws_path = ws_scheme + '://' + newHost + `/ws/${clientId}?llm_model=${selectedModel}?token=${token}`;
```

Comment:
should be ?llm_model=${selectedModel}&token=${token} 
use & to separate parameters